app/services/discovery_engine/sync/operation.rb

```diff
@@ -1,12 +1,13 @@
 module DiscoveryEngine::Sync
   class Operation
-    def initialize(content_id, payload_version: nil)
+    def initialize(content_id, payload_version: nil, client: nil)
```

Comment:
Should `content_id` have a default value of `nil` here? I feel like `content_id` should be a required value as so much relies upon it's existence.